.github/workflows/example-android-build-check.yml

```diff
@@ -47,19 +47,6 @@ jobs:
           distribution: 'zulu'
           java-version: 17
 
-      # TODO: Rework caching in this CI for monorepo setup.
```

Comment:
Let's keep such comment here instead:

`# TODO: Add caching for node_modules and artifacts that will work with monorepo setup.`